templates/admin/stacktrace-row.tmpl

```diff
@@ -17,7 +17,10 @@
 		</div>
 		<div>
 			{{if or (eq .Process.Type "request") (eq .Process.Type "normal")}}
-				<a class="delete-button icon" href="" data-url="{{.root.Link}}/cancel/{{.Process.PID}}" data-id="{{.Process.PID}}" data-name="{{.Process.Description}}">{{svg "octicon-trash" 16 "text-red"}}</a>
+				<a class="link-action" data-url="{{.root.Link}}/cancel/{{.Process.PID}}"
+					data-modal-confirm-header="{{ctx.Locale.Tr "admin.monitor.process.cancel"}}"
```

Comment:
```suggestion
				<a class="link-action"
				  data-url="{{.root.Link}}/cancel/{{.Process.PID}}"
					data-modal-confirm-header="{{ctx.Locale.Tr "admin.monitor.process.cancel"}}"
```

Reply:
I have no motivation to change. The existing code looks better to me.

Reply:
I can't approve that style, either all properties should be on same line, or all on separate lines.

Reply:
> I can't approve that style, either all attributes should be on same line, or all on separate lines.

Well, you have your style opinion, I have mine. I will ask others to review.